boot/freeldr/freeldr/CMakeLists.txt

```diff
@@ -122,6 +122,12 @@ if(ARCH STREQUAL "i386")
         arch/i386/i386idt.c
         disk/scsiport.c)
 
+    if(SARCH STREQUAL "pc98" OR SARCH STREQUAL "xbox")
+        # These machine types require built-in font
+        list(APPEND FREELDR_ARC_SOURCE
+            ui/vgafont.c)
```

Comment:
Please no, the UI interface has nothing to do with the specific hardware font: it should go back somewhere into arch/ .

Reply:
Note that @bragin also used it in his OLPC branch - https://github.com/reactos/reactos/commit/33c7a790d37515677d262e08c69497531682d994#diff-359e2b239afce432c3965a0dfa65a5f5R156

The font itself can be used on any hardware that allows a framebuffer or a similar video output technology, thus I doubt about `arch/` directory. Any suggestions? :)

Reply:
@HBelusca ok I moved it.